README.md

```diff
@@ -109,5 +109,18 @@ The linter provides the possibility to implement these customized specifications
 
 More information about how this feature works is documented [here](docs/customize_specifications.md)
 
+## Pre-Commit
```

Comment:
very very very very minor, `pre-commit` tends to be stylized lowercase

Reply:
/o\ both characters lower case

(that's me hiding, I shouldn't have said anything -- sorry for the inconvenience)